scripts/generate_aviti_run_manifest.py

```diff
@@ -323,13 +323,17 @@ def make_manifest(
         [
             "[SETTINGS]",
             "SettingName, Value",
+            # Mismatch thresholds set to 0 to allow identical I1 xor I2
```

Comment:
This does not only allow identical indicies, but enforce it during demultiplexing. When not necessary, we would still throw away reads with 1-2 mismatches to indices in the sample sheet.